### DISCUSSION TITLE
[8주차] 온라인 스터디 (면접)

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#59] [General]
### 진행순서
~ 20:15 카훗`Kahoot`으로 입풀기
~ 21:00 1:N 모의면접 진행
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 🙂
* 좋았던 점
* 건의사항
* 느낀점

**Reply by leeejuhyeong** [reply]
나도 잘하고싶다

**Reply by ChaerinYu** [reply]
다들 바쁘신 와중에도 참여해주셔서 감사합니다 ㅎㅎ 😎👍

**Reply by KJY97** [reply]
다음에는 잘할게요.. 감사합니다!

**Reply by ChaerinYu** [reply]
@Kodo92 동호님도 출석

**Reply by doho-ho** [reply]
c출석완료 담주 발표할거 정햇어요
